public/app/features/search/service/searcher.ts

```diff
@@ -1,11 +1,12 @@
 import { BlugeSearcher } from './bluge';
+import { SQLSearcher } from './sql';
 import { GrafanaSearcher } from './types';
 
 let searcher: GrafanaSearcher | undefined = undefined;
 
 export function getGrafanaSearcher(): GrafanaSearcher {
   if (!searcher) {
-    searcher = new BlugeSearcher();
+    searcher = true ? new SQLSearcher() : new BlugeSearcher();
```

Comment:
@axelavargas -- Ideally we can make the feature toggle here and always use the same (UID based) components.

Still a bit more hardening to do, but ballpark it looks like this approach will work!